perses/tests/utils.py

```diff
@@ -777,14 +787,16 @@ def validate_endstate_energies(topology_proposal, htf, added_energy, subtracted_
         energy_comps = compute_potential_components(context)
         for name, force in energy_comps:
            print("\t\t\t{}: {}".format(name, force))
-        print(f'added forces:{sum([energy for name, energy in energy_comps])}')
-        print(f'rp: {rp}')
+        _logger.debug(f'added forces:{sum([energy for name, energy in energy_comps])}')
+        _logger.debug(f'rp: {rp}')
         del context, integrator
 
-    #print(f"added_energy: {added_energy}; subtracted_energy: {subtracted_energy}")
     nonalch_zero_rp, alch_zero_rp, alch_one_rp, nonalch_one_rp = rp_list[0], rp_list[1], rp_list[2], rp_list[3]
-    assert abs(nonalch_zero_rp - alch_zero_rp + added_energy) < ENERGY_THRESHOLD, f"The zero state alchemical and nonalchemical energy absolute difference {abs(nonalch_zero_rp - alch_zero_rp + added_energy)} is greater than the threshold of {ENERGY_THRESHOLD}."
-    assert abs(nonalch_one_rp - alch_one_rp + subtracted_energy) < ENERGY_THRESHOLD, f"The one state alchemical and nonalchemical energy absolute difference {abs(nonalch_one_rp - alch_one_rp + subtracted_energy)} is greater than the threshold of {ENERGY_THRESHOLD}."
+
+    ratio = (nonalch_zero_rp - alch_zero_rp + added_energy) / (0.5*(nonalch_zero_rp + alch_zero_rp + added_energy))
```

Comment:
i feel dumb, but can you explain the motivation for using `0.5`?

Reply:
I think it's to make the relative error computation symmetric?
Shouldn't we also take the absolute value to compute a relative error?

Reply:
It does use abs on line 797 and 799 but I can move it up if that's more legible. Also the 0.5 isn't necessary as we can just change the threshold by a factor of two it doesn't really matter